tools/meson-post-install.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+datadir=$1
+bindir=$2
+pkgconfdir=$3
+pkglibdir=$4
+localstatedir=$5
+
+[ -n "$DESTDIR" ] && DESTDIR=${DESTDIR%%/}/
+
+mv "${DESTDIR}${datadir}/bash-completion/completions/nmcli-completion" \
+   "${DESTDIR}${datadir}/bash-completion/completions/nmcli"
+
+for alias in nmtui-connect nmtui-edit nmtui-hostname; do
+    ln -sf nmtui "${DESTDIR}${bindir}/$alias"
+done
```

Comment:
if tui isn't enabled, these symlinks shouldn't be installed.

Reply:
Fixed.